firedrake/utility_meshes.py

```diff
@@ -198,6 +198,109 @@ def PeriodicUnitIntervalMesh(ncells):
     return PeriodicIntervalMesh(ncells, length=1.0)
 
 
+def OneElementThickMesh(ncells, L):
```

Comment:
Ok,  if you don't mind cluttering with if  statements.

Cjc

On Wednesday, 8 June 2016, David A. Ham <notifications@github.com> wrote:

> In firedrake/utility_meshes.py
> <https://github.com/firedrakeproject/firedrake/pull/803#discussion_r66287634>
> :
>
> > @@ -198,6 +198,109 @@ def PeriodicUnitIntervalMesh(ncells):
> >      return PeriodicIntervalMesh(ncells, length=1.0)
> >
> >
> > +def OneElementThickMesh(ncells, L):
>
> I agree with dorugeber
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/firedrakeproject/firedrake/pull/803/files/0bbb07b2ed6725ca0ceca05250a2adc5380d3632#r66287634>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AF0q8p7D93sjITiF8gYlNblt8cpV9FEbks5qJurfgaJpZM4Iw-Dk>
> .
>


-- 
http://www.imperial.ac.uk/people/colin.cotter

www.cambridge.org/9781107663916